lib/cocoapods/generator/xcconfig/xcconfig_helper.rb

```diff
@@ -91,7 +99,7 @@ def self.add_spec_build_settings_to_xcconfig(consumer, xcconfig)
         #
         def self.add_framework_build_settings(framework_path, xcconfig, sandbox_root)
           name = File.basename(framework_path, '.framework')
-          dirname = '$(PODS_ROOT)/' + framework_path.dirname.relative_path_from(sandbox_root).to_s
+          dirname = '${PODS_ROOT}/' + framework_path.dirname.relative_path_from(sandbox_root).to_s
```

Comment:
When I ran the `install_vendored_dynamic_framework` integration spec, it initially spit out the following 
```
FRAMEWORK_SEARCH_PATHS = FRAMEWORK_SEARCH_PATHS = $(inherited) "${PODS_ROOT}/Realm/realm-swift-0.94.1/ios/swift-1.2" "$(PODS_ROOT)/Realm/realm-swift-0.94.1/ios/swift-1.2" 
```

I took care of the duplication, but figured we wanted to go with curly braces (since the duplication my doing). Happy to change it back.